evalml/data_checks/invalid_targets_data_check.py

```diff
@@ -89,6 +89,12 @@ def validate(self, X, y):
                                                      data_check_name=self.name,
                                                      message_code=DataCheckMessageCode.TARGET_BINARY_NOT_TWO_UNIQUE_VALUES,
                                                      details=details).to_dict())
+        if self.problem_type == ProblemTypes.MULTICLASS and value_counts.min() <= 1:
```

Comment:
This works, but maybe it'd be more useful for users if we give them all of the classes that don't have at least 2 instances? For example if multiple classes just have one sample, this would only return one of those classes and the user would have to remove to see the other class.

Reply:
Added.